src/structures/MessageEmbed.js

```diff
@@ -231,7 +231,7 @@ class MessageEmbed {
   }
 
   /**
-   * The accumulated length for the embed title, description, fields and footer text
+   * The accumulated length for the embed title, description, fields, footer text and author name
```

Comment:
Oxford comma ¯\_(ツ)_/¯
```suggestion
   * The accumulated length for the embed title, description, fields, footer text, and author name
```